Assets/MRTK/Providers/XRSDK/Controllers/GenericXRSDKController.cs

```diff
@@ -6,6 +6,7 @@
 using Unity.Profiling;
 using UnityEngine;
 using UnityEngine.XR;
+using Unity.XR.Oculus;
```

Comment:
This should be in the `OculusXRSDKTouchController` definition, similar to how the WMRUsages are in the WMR-specific class. That way, this class stays as generic and cross-platform as possible!